styles/base.css

```diff
@@ -157,6 +185,35 @@ h4,
     line-height: 130%;
 }
 
+.h-sm,
+.blog_post h2 {
+    font-size: var(--h-sm-font-size);
+    font-weight: var(--h-sm-font-weight);
+    line-height: var(--h-sm-line-height);
+}
+
+.secondary-xs,
```

Comment:
The `secondary` refers to the fact that this has the Sans Serif font family. In this app's case, the Primary is Monospace and the Secondary is San Serif (Roboto).

The `XS` refers to font size. `XS` is 1rem/16px.

Reply:
Should we maybe just be explicit with `sans` vs. `monospace`?

Reply:
Gonna try this: To be extra clear -

`primary-monospace`
`secondary-sans-serif`